nntrainer/utils/parse_util.cpp

```diff
@@ -136,6 +137,8 @@ unsigned int parseType(std::string ll, InputType t) {
 
   switch (t) {
   case TOKEN_LOSS:
+    if (ll.empty())
```

Comment:
1. I think ll never goes to empty even the loss is omitted. 
Instead it would be a "none".
Check line 62 at model_loader.cpp

2. Also would it be better to declare a none at the beginning in loss_string, LossType?
Then these 2 line could be deleted :)

Reply:
will do.

Reply:
removed the empty check. `none` is already part of `loss_string`.